vis/js/templates/headingtitles/BasicTitle.jsx

```diff
@@ -0,0 +1,9 @@
+import React, { Fragment } from "react";
+
+const BasicTitle = ({ title }) => {
+  // dangerously Setting the Inner HTML because of e.g. Covis, where the basic title is HTML
+  // the span must be here so the dangerouslySetInnerHTML works
+  return <span dangerouslySetInnerHTML={{ __html: title }}></span>;
```

Comment:
similarly here, we probably want to try and separate the html and the messages here. This content should really be a new component/template; this could of course be done in a followup